webapp/loggedin/appSideBar/appSideBarModules.js

```diff
@@ -81,9 +81,9 @@ const AppSideBarModules = ({ pathname, surveyInfo, sideBarOpened }) => (
   <div className="app-sidebar__modules">
     {
       modulesHierarchy.map((module, i) => (
-        <React.Fragment>
+        <React.Fragment key={i}>
```

Comment:
Why not key={module.key}?

Reply:
because it's the same.. i is shorter... but i updated it in the next commit